.github/workflows/test-workflows.yml

```diff
@@ -13,21 +13,21 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v3.5.3
         with:
           path: n8n
 
       - name: Checkout workflows repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v3.5.3
         with:
           repository: n8n-io/test-workflows
           path: test-workflows
 
-      - uses: pnpm/action-setup@v2.2.4
+      - uses: pnpm/action-setup@v2.4.0
         with:
-          version: 8.6.1
+          version: 8.6.12
```

Comment:
Should we pin the pnpm version everywhere else as well?

Reply:
this is the only place we need it. elsewhere the version is automatically picked from `package.json`. I'm going to soon change this workflow to also pick the pnpm version from `package.json`, so this exception will hopefully go away soon.